src/components/TextInputWithPrefix/index.js

```diff
@@ -39,6 +39,10 @@ const TextInputWithPrefix = props => (
                     styles.textInputWithPrefix.textInput,
                     styles.noOutline,
                 ]}
+
+                // Align Text to left by default to override align to right by default in RTL which is not supported.
```

Comment:
```suggestion
                // By default, align input to the left to override right alignment in RTL mode which is not yet supported in the App.
```
let's follow this comment everywhere. Thanks.

Reply:
ok , done .